test/reporters/xunit.spec.js

```diff
@@ -194,15 +194,13 @@ describe('XUnit reporter', function() {
       it('should call write with line', function() {
         stdoutWrite = process;
         process = false; // eslint-disable-line no-native-reassign, no-global-assign
-        var cachedConsoleLog = console.log;
-        console.log = function(string) {
-          stdout.push(string);
-        };
 
+        var cachedPrintln = reporters.Base.log;
+        reporters.Base.log = stdout.push.bind(stdout);
         var xunit = new XUnit({on: function() {}, once: function() {}});
         xunit.write.call({fileStream: false}, expectedLine);
+        reporters.Base.log = cachedPrintln;
```

Comment:
Don't see why any changes should be made here... the PR was addressing user-related squelching of output, not modifying unrelated tests.

cachedPrintln?

Reply:
So the test relies on controlling console.log. With my base changes it fails.
Do you see a better way of doing this?

Will rename `cachedConsoleLog`.